### PR TITLE
Add a method to show loclist

### PR DESCRIPTION
**Pull request by himansingh241**

### CODE REVIEW

File: lua/telescope/builtin.lua [hunk 1]
```diff
@@ -112,6 +112,13 @@ end
 
 builtin.quickfix = function(opts)
   local locations = vim.fn.getqflist()
+  if next(locations) == nil then
+    locations = vim.fn.getloclist(0)
+    for _, value in pairs(locations) do
+      value.filename = vim.fn.bufname("%")
```
Comment (tjdevries) [inline lua/telescope/builtin.lua:118]: vim.api.nvim_buf_get_name(0) would be better

File: lua/telescope/builtin.lua [hunk 2]
```diff
@@ -112,6 +112,13 @@ end
 
 builtin.quickfix = function(opts)
```
Comment (tjdevries) [inline lua/telescope/builtin.lua:113]: Can you add an opt for `loclist_fallback`?

Comment (tjdevries) [reply lua/telescope/builtin.lua:113]: Or.... hmm, maybe what we want is just two different functions.

Comment (tjdevries) [reply lua/telescope/builtin.lua:113]: They mean very different things.

Comment (himansingh241) [reply lua/telescope/builtin.lua]: Done!

**Comment by himansingh241** [comment]
I can add the info about this in README.md if you think the implementation is okay.

File: lua/telescope/builtin.lua [hunk 3]
```diff
@@ -126,6 +126,28 @@ builtin.quickfix = function(opts)
   }):find()
 end
 
+builtin.loclist = function(opts)
+  local locations = vim.fn.getloclist(0)
+
+  for _, value in pairs(locations) do
+    value.filename = vim.api.nvim_buf_get_name(0)
```
Comment (tjdevries) [inline lua/telescope/builtin.lua:133]: probably only need to get this once. Just for perf that doesn't matter.

**Pull request closed without merging**